### ISSUE TITLE
Avoid using deprecated APIs

### ISSUE THREAD

**Issue by fantix** [fantix/gino#452]
Running tests, these warning messages are seen:

SQLAlchemy:

```
  <string>:2: SADeprecationWarning: The text.bindparams parameter is deprecated and will be removed in a future release.  Please refer to the TextClause.bindparams() method.
```

aiohttp:

```
  /Volumes/Home/fantix/PycharmProjects/gino/gino/ext/aiohttp.py:144: DeprecationWarning: loop property is deprecated
    loop=app_.loop,
```

We should move away from the deprecated API.

**Comment by teamhide** [comment]
It still happens. When will it be fixed?

**Comment by wwwjfy** [comment]
It's fixed, but in master, not released

**Comment by wwwjfy** [comment]
@teamhide with the latest released 0.8.3, the warning should no longer appear.
